docs/package.json

```diff
@@ -12,7 +12,7 @@
   "private": true,
   "scripts": {
     "start": "npm run browser:development",
-    "browser:development": "webpack-dev-server --config webpack-dev-server.config.js --progress --colors --inline --host=",
+    "browser:development": "webpack-dev-server --config webpack-dev-server.config.js --progress --colors --inline",
```

Comment:
When using ` --host=` I have a warning in the console saying the hot reloading end point can't be reached.
I haven't investigated. I just though it would be safer to revert.